changelogs/fragments/ansible-test-coverage-update.yml

```
@@ -0,0 +1,7 @@
+---
+
+bugfixes:
```

This should probably be
```suggestion
minor_changes:
```